src/connector.js

```diff
@@ -1030,7 +1044,11 @@ function updateResultListState( newState ) {
 			let description = "";
 			let printableUri = encodeURI( result.printableUri );
 			printableUri = printableUri.replaceAll( '&' , '&amp;' );
+			printableUri = printableUri.replaceAll( '%252F' , '/' );			
+			printableUri = printableUri.replaceAll( "%252C" , "," ); // handle coma							   
 			let clickUri = encodeURI( result.clickUri );
+			clickUri = clickUri.replaceAll( "%252C" , "%2C" );  // handle coma
+			clickUri = clickUri.replaceAll( "%252F" , "%2F" );  // handle slash							   
```

Comment:
@igkislev Can you please provide some queries (and search pages if needed) to test these fixes you are introducing? I want to run functional testing. Thanks!

Reply:
I'll also check for regular Canada.ca results, as well as Elections link that had a previous track record for problematic HTML entities.